pyproject.toml

```diff
@@ -20,6 +20,7 @@ dependencies = [
     "mlx==0.7.*; sys_platform == 'darwin'",
     "numpy==1.26.3",
     "requests==2.31.0",
+    "networkx",
```

Comment:
we may not want to have this - people wanting to use the conversion functionalities will have to install it, but it shouldn't be a dependency of ours. We can add it to the optional dependencies.

Reply:
I think we can add networkx to the test dependencies?
Or we have to use `@pytest.mark.skipif(not networkx_installed, reason="networkx is not installed")` like for `test_to_networkx()`.